.changeset/honest-files-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@monokle/components": minor
```

Comment:
same here, it should be a patch release

Reply:
for me it makes sense if it's a minor release, since you're adding a new component and it's not just a patch to fix something 🤔